### PR TITLE
Remove Analytics Catalyst tests.

### PR DESCRIPTION
**Pull request by ryanwilson**
By specifying iOS in the package manifest, it includes Analytics but the
Analytics xcframework doesn't have a Catalyst slice.

#no-changelog

### CODE REVIEW

File: .github/workflows/spm.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ jobs:
 
   cron-only:
     # Don't run on private repo.
-    if: github.event_name == 'schedule' && github.repository == 'Firebase/firebase-ios-sdk'
+    #    if: github.event_name == 'schedule' && github.repository == 'Firebase/firebase-ios-sdk'
```
Comment (paulb777) [inline .github/workflows/spm.yml:32]: Restore after testing.

**Review by paulb777 — APPROVED** [review]
Thanks! LGTM after green and cron restore.

File: Package.swift [hunk 2]
```diff
@@ -801,8 +801,6 @@ let package = Package(
       dependencies: [
         "FirebaseAuth",
         "FirebaseABTesting",
-        .target(name: "FirebaseAnalytics",
```
Comment (paulb777) [inline Package.swift:804]: Same change needed for the swift test

Comment (ryanwilson) [reply Package.swift:804]: Good catch, done.

File: .github/workflows/spm.yml (hunk 1 above)
Comment (ryanwilson) [reply .github/workflows/spm.yml]: Done.

**Comment by paulb777** [comment]
Merging to get cron fixed for tonight.

**Pull request merged**